src/ArduinoGraphics.cpp

```diff
@@ -124,6 +124,51 @@ void ArduinoGraphics::noStroke()
   _stroke = false;
 }
 
+void ArduinoGraphics::ellipse(int x, int y, int width, int height)
+{
+  if (!_stroke && !_fill) {
+    return;
+  }
+
+  float theta;
+  int r1 = width / 2;
+  int r2 = height / 2;
+  for (int angle = 0; angle < 360; angle += 1) {
+    theta = angle * 3.14 / 180;
+    int xi = r1 * cos(theta);
+    int yi = r2 * sin(theta);
+    if (_stroke) {
+        // stroke
```

Comment:
Please fix the indentation. The rest of the code (and even much of yours) using a two space indent, but then you used some four space indents, which makes the code hard to read.

Reply:
fixed.